### ISSUE TITLE
`amount` in `api/v1/users/me/top-up` should be provided in request body  

### ISSUE THREAD

**Issue by arneallwardt** [53845714nF/EventTom#64]
see #56 

<img width="1222" alt="Screenshot 2024-12-31 at 11 39 11" src="https://github.com/user-attachments/assets/fa62118a-8382-449a-875f-519d0db20e8f" />

**Comment by LuchsHub** [comment]
Yeah, I'll look over all the methods I created so far and update all the ones I falsely created with query params later today.
